CHANGELOG.md

```diff
@@ -5,84 +5,100 @@
 decorator-transforms 2.2.2 (patch)
 
 #### :bug: Bug Fix
-* `decorator-transforms`
-  * [#35](https://github.com/ef4/decorator-transforms/pull/35) Remove unintentional change that was part of #33 ([@ef4](https://github.com/ef4))
+
+- `decorator-transforms`
```

Comment:
I probably wouldn't have prettier apply to the changelog since it's auto-generated 🤔

Reply:
rogerroger.gif